common/logger/src/lib.rs

```diff
@@ -49,18 +49,25 @@ pub mod prelude {
 
 pub use simple_logger::{set_simple_logger, set_simple_logger_prefix};
 
-/// Creates and sets default global logger.
+/// Creates and sets the default global logger.
```

Comment:
what does "default" mean, it seems there is only one global logger.

Reply:
There is only one global logger, but "default" here means at the default filter level. I've updated the comment to be more explicit.